drivers/gles3/shaders/tonemap.glsl

```diff
@@ -156,6 +156,17 @@ vec3 tonemap_aces(vec3 color, float white) {
 }
 
 vec3 tonemap_reinhard(vec3 color, float white) {
+	// Ensure color values are pozitive
+	// They can be negative in the case of negative lights, which leads to undesired behaviour
+	if (color.r < 0.0f)
```

Comment:
Usually, for that kind of computation, we use max(...), it makes the code simpler.

So basically: 
```
color.r = max(color.r, 0.0f)
color.g = max(color.g, 0.0f)
color.b = max(color.b, 0.0f)
```

Reply:
Or even better in GLSL:
```
color = max(vec3(0.0), color);
```

Reply:
Done !